src/perf/bin/appmain.cpp

```diff
@@ -41,7 +45,24 @@ QuicUserMain(
     QUIC_EVENT StopEvent;
     QuicEventInitialize(&StopEvent, true, false);
 
-    QUIC_STATUS Status = QuicMainStart(argc, argv, &StopEvent, SelfSignedConfig);
+    uint8_t ServerMode = 0;
+    TryGetValue(argc, argv, "ServerMode", &ServerMode);
+
+    QUIC_STATUS Status;
+    QUIC_DATAPATH* Datapath;
+    QUIC_DATAPATH_BINDING* Binding;
+
+    if (ServerMode) {
+        QuicDataPathInitialize(256, DatapathReceiveUserMode, DatapathUnreachable, &Datapath);
+        QUIC_ADDR LocalAddress;
+        QuicZeroMemory(&LocalAddress, sizeof(LocalAddress));
+        QuicAddrSetPort(&LocalAddress, 9999);
+        QuicAddrSetFamily(&LocalAddress, AF_INET);
```

Comment:
No need to zero init since you're setting port and family. BTW, set family to AF_UNSPEC. No need to hard code to V4. Never know when we want to use V6 instead.